analysis_options.yaml

```diff
@@ -1 +1,6 @@
 include: package:solid_lints/analysis_options.yaml
+
+linter:
+  rules:
+    sort_constructors_first: true
```

Comment:
Any particular reason you're overriding this?

The lint rules added to the project are the standard for our company.